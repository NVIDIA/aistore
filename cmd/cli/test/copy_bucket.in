ais bucket create ais://$BUCKET_1 ais://$BUCKET_2 | awk '{print $1 " " $2}'

for d1 in {0..4}; do for d2 in {0..100}; do mkdir -p /tmp/rdir/test$d1/$d2 && echo -n $RANDOM > /tmp/rdir/test$d1/$d2/test.txt; done; done
ais object put "/tmp/rdir/test{0..4}/{0..100}/test.txt" ais://$BUCKET_1 -y // IGNORE

ais bucket cp aiss://$BUCKET_1/ ais://$BUCKET_2/ // FAIL "invalid backend provider "aiss""
ais bucket cp ais://$BUCKET_1/ aiss://$BUCKET_2/ // FAIL "invalid backend provider "aiss""
ais bucket cp ais://$BUCKET_1 ais://$BUCKET_2 --force

ais job wait copy-bck ais://$BUCKET_2
ais show job copy-bck ais://$BUCKET_1 --all
ais show job copy-bck
ais show job copy-bck ais://$BUCKET_3 // FAIL ais://$BUCKET_3

rm -rf /tmp/rdir // IGNORE
