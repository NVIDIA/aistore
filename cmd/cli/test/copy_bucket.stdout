"ais://$BUCKET_1" created
"ais://$BUCKET_2" created
^Copying bucket ais://$BUCKET_1 => ais://$BUCKET_2. To monitor.*$
^Waiting for.*$
^copy-bucket.*done.$
^No running.*$
^No running.*$
